src/main/java/org/pac4j/vertx/cas/VertxSharedDataLogoutHandler.java

```diff
@@ -71,9 +71,7 @@ public void destroySession(WebContext context) {
         final String sessionId = getSessionId(ticket);
```

Comment:
I didn't remember there was specific classes for CAS logout in Vertx, but indeed, you have worked on that. In pac4j v1.9.2, I'm working on making CAS logout (back channel, front channel) works easily and everywhere. Everything should be backward compatible with your current implementations as you inherit from `NoLogoutHandler`. We'll talk about that later on.